ibm_catalog.json

```diff
@@ -218,11 +225,15 @@
               "name": "testing-deploy-arch-ibm-observability",
               "id": "daf2cf91-2384-4205-b7f4-621299a4896d-global",
               "version": "^1.3.0",
-              "flavors": ["standard"],
+              "flavors": [
+                "standard"
+              ],
               "catalog_id": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3",
               "optional": true,
               "on_by_default": true,
-              "ignore_auto_referencing": ["*"],
+              "ignore_auto_referencing": [
```

Comment:
This is not required. Asked [here](https://ibm-cloudplatform.slack.com/archives/CG06FQ31P/p1747395625959889)